### PR TITLE
Fix problems when batch size is small

### PR DESCRIPTION
**Pull request by xzdandy**
- append ignores index 
- sort returned batch in integration test cases
- Not yield empty batch

### CODE REVIEW

File: src/executor/seq_scan_executor.py [hunk 1]
```diff
@@ -51,4 +51,5 @@ def exec(self) -> Iterator[Batch]:
                 batches = [expr.evaluate(batch) for expr in self.project_expr]
                 batch = Batch.merge_column_wise(batches)
 
-            yield batch
+            if not batch.empty():
```
Comment (gaurav274) [inline src/executor/seq_scan_executor.py:54]: Do we need this check? Yielding an empty batch might make things easy for the caller

File: src/executor/plan_executor.py [hunk 2]
```diff
@@ -106,7 +107,11 @@ def execute_plan(self):
             execution_tree.exec()
         else:
             for batch in execution_tree.exec():
-                output_batches += batch
+                if not batch.empty():
```
Comment (gaurav274) [inline src/executor/plan_executor.py:110]: We can move this check to the _add_ function

File: src/executor/seq_scan_executor.py (hunk 1 above)
Comment (xzdandy) [reply src/executor/seq_scan_executor.py]: I move this check to when we apply predicate, and it also works.

File: src/executor/plan_executor.py (hunk 2 above)
Comment (xzdandy) [reply src/executor/plan_executor.py]: Move to the add function.

**Pull request merged**